### PR TITLE
l2 network switch

### PR DESCRIPTION
**Pull request by vbstreetz**

### CODE REVIEW

File: sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx [hunk 1]
```diff
@@ -2,51 +2,84 @@ import { FC, useState } from 'react';
...
-import { addOptimismNetworkToMetamask } from '@synthetixio/optimism-networks';
-import { NetworkId } from '@synthetixio/js';
+import { isL2State, isMainnetNetworkState } from 'store/wallet';
+import Tippy from '@tippyjs/react';
+// import { addOptimismNetworkToMetamask } from '@synthetixio/optimism-networks';
+// import { NetworkId } from '@synthetixio/js';
 
 type NetworksSwitcherProps = {};
 
+const OVM_NETWORKS: Map<number, any> = new Map([
```
Comment (evgenyboxer) [inline sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx:12]: can we have it typed instead of `any`?

Comment (vbstreetz) [reply sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx]: awesome. done 👍🏽

File: containers/Connector/Connector.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ const useConnector = () => {
 			const useOvm = getIsOVM(networkId);
 
 			synthetix.setContractSettings({
-				networkId: useOvm ? 1 : networkId,
```
Comment (JChiaramonte7) [inline containers/Connector/Connector.tsx:53]: Maybe a util function for networkID because we reuse this a few times

**Review by JChiaramonte7 — APPROVED** [review]

File: containers/Connector/Connector.tsx (hunk 2 above)
Comment (vbstreetz) [reply containers/Connector/Connector.tsx:53]: Say more, the edit here actual removes that conditional :)

Comment (JChiaramonte7) [reply containers/Connector/Connector.tsx:53]: You're right. I'm not sure why I was looking at removed lines. Everything looks good!

**Pull request merged**